setup.py

```diff
@@ -107,5 +107,5 @@ def find_rel_test():
     provides=["eb"] + easybuild_packages,
     test_suite="test.framework.suite",
     zip_safe=False,
-    install_requires=["vsc-base >= 2.2.4"],
+    install_requires=["vsc-base >= 2.2.4", "PyYAML >= 3.11"],
```

Comment:
+1.

I happen to play with a another code tree these days which relies on yaml, and basically it proved to be a major pain on systems with python2.6 as a default, since it is routinely just not there (even if, it is easily obtainable in one way or another)